sources/AppBundle/Controller/LeadController.php

```diff
@@ -59,4 +72,35 @@ public function becomeSponsorLatestAction()
 
         return new RedirectResponse($this->generateUrl('sponsor_leads', ['eventSlug' => $event->getPath()]));
     }
+
+    /**
+     * @param Event $event
+     * @param Lead $lead
+     */
+    private function sendMailToTeamSponsor(Event $event, Lead $lead)
+    {
+        $subject = sprintf('%s - Nouvelle demande de dossier de sponsoring', $event->getTitle());
+
+        $content =
+            sprintf(
+                "Une nouvelle demande de dosssier de sponsoring vient d'être effectuée sur le site. Voici les informations saisies :
+                
+                - Société: %s
+                - Nom: %s
+                - Prénom: %s
+                - Email: %s
+                - Téléphone: %s
+                - Website: %s
```

Comment:
```suggestion
                - Site web: %s
```

C'est juste un tout petit point de détail, est-ce qu'il ne faudrait pas mettre cela en français vu que le reste est français ?